lib/Sema/TypeCheckDecl.cpp

```diff
@@ -6061,6 +6061,19 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
           new (TC.Context) OverrideAttr(SourceLoc()));
     }
 
+    // If the overridden method is declared in a Swift Class Declaration,
+    // dispatch will use table dispatch. If the override is in an extension
+    // warn, since it is not added to the class vtable.
+    // FIXME: Only warn if the extension is in another module, and if
```

Comment:
Can you put a new comment in between the FIXME and the rest of the comment.

So this:

// Foo
// Bar
// FIXME:

Becomes

// Foo
// Bar
//
// FIXME:

This optimizes the FIXME for the reader who is quickly scanning the file by making it more obvious.